<launch>
  <node pkg="bw_ros_driver" type="bw_node" name="bw_node" output="screen">
    <rosparam command="load" file="$(find bw_ros_driver)/config/params.yaml"/>
  </node>
</launch>
<!-- <param name="port" value="/dev/ttyUSB0"/>
<param name="baud" value="115200"/>
<param name="frame_id" value="imu_link"/>
<param name="topic" value="/imu/data"/>
<param name="debug" value="true"/>
<param name="mag_unit" value="gauss"/>
<param name="cov_orientation" value="-1"/>
<param name="cov_angular_velocity" value="-1"/>
<param name="cov_linear_acceleration" value="-1"/> -->